ports/libdjinterop/portfile.cmake

```
@@ -1,18 +1,18 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO xsco/libdjinterop
-    REF ${VERSION}
+    REF "${VERSION}"
     SHA512 7becb83ab62412b3d437ddee23b248a697b162f6b8a64070cd8a9782a4fce7726baaf12ea193b8e21bcf561a00039ab1ae1f04d00e6cbe8344ec19751779db14
     HEAD_REF master
 )
 
 vcpkg_cmake_configure(SOURCE_PATH "${SOURCE_PATH}")
 vcpkg_cmake_install()
 
-vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/DjInterop)
+vcpkg_cmake_config_fixup(PACKAGE_NAME djinterop CONFIG_PATH lib/cmake/DjInterop)
```

```suggestion
vcpkg_cmake_config_fixup(PACKAGE_NAME DjInterop CONFIG_PATH lib/cmake/DjInterop)
```
Not sure if case sensitivity is an issue but it is better to have it correct on Linux and macOs

---

@daschuer Cf. https://github.com/microsoft/vcpkg/pull/32312#discussion_r1247452890: lower-case *is correct for CMake* and (probably) *desired for vcpkg*.

---

OK, I can confirm the lower case version works.

---

Usage passed on `x64-linux`.